docs/features/bgp-integration/route-advertisements.md

```diff
@@ -0,0 +1,620 @@
+# Route Advertisements
+
+## Introduction
+
+The Route Advertisements feature introduces BGP as a supported routing protocol
+with OVN-Kubernetes enabling the integration into different BGP user
+environments. The extent of the Route Advertisements feature and corresponding
+API allows importing routes from BGP peers into OVN pod networks as well as
+exporting pod network and egress IP routes to BGP peers. Both default pod
+network as well as primary Layer 3 and Layer 2 cluster-user-defined networks
+(CUDNs) are supported.
+
+## Prerequisites
+
+- [FRR-k8s](https://github.com/metallb/frr-k8s)
+
+## Motivation
+
+There are multiple driving factors which necessitate integrating BGP into
+OVN-Kubernetes (for purposes of this documentation, the external, physical
+network of the cluster which a user administers will be called the “provider
+network”):
+
+- Importing Routes from the Provider Network: Today there is no API for a user
+to be able to configure routes into OVN. In order for a user to change how
+egress traffic is routed, the user leverages local gateway mode. This mode
+forces traffic to hop through the Linux networking stack, and there a user can
+configure routes inside the host to control egress routing. This manual
+configuration would need to be performed and maintained across nodes and VRFs
+within each node.
+
+- Exporting Routes into the Provider Network: There exists a need for provider
+networks to learn routes directly to services and pods today in Kubernetes. One
```

Comment:
Shall we metion services here? We don't support advertising routes to service.

Reply:
So this phase a general introductory phrase (I blamelessly took from the BGP enhancement) and is talking in context of `in kubernetes`, referring to things like what is implemented in MetalLB to expose services with BGP.